### ISSUE TITLE
Pages stop loading

### ISSUE THREAD

**Issue by ColorCookie-dev** [KotatsuApp/Kotatsu#256]
### Brief summary

When reading a manga in __webtoons__ layout, after sometime _(usually within 5 minutes)_, suddenly an upcoming page is stuck on loading and does not load even after waiting (I usually don't wait for more than 1 minute).

However after closing the manga and opening it again (reopening manga not the whole app) the same page loads instantly and everything seems correct. Still after sometime it happens again. It repeats endlessly.

I usually use webtoon layout, so I don't know if it fails in other layout.

I've noticed this bug after the UI change recently.

### Steps to reproduce

Steps to reproduce (as far as it works on my phone):
1. Open kotatsu
2. Open a manhua in webtoon layout
3. Read for sometime, with normal speed
4. After sometime, usually within 5 minutes, a page comes up which is stuck loading.

### Kotatsu version

4.0.2

### Android version

12.0

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.

**Comment by ColorCookie-dev** [comment]
New development: I've noticed now that instead of reopening the manga, if I just scroll up a good amount, and scroll back down, it loads quickly.

**Comment by ColorCookie-dev** [comment]
Why is the author ignoring this issue? Is nobody experiencing the same thing? The bug is really annoying, I'm considering downgrading to version 3.5

**Comment by Koitharu** [comment]
Sorry, I have a small lack of time. Also I cannot reproduce this(

**Comment by Koitharu** [comment]
Try to change "pages preloading" option in the reader settings

**Comment by ColorCookie-dev** [comment]
I changed pages preloading option, however it still happens, I changed it to "Always" and "Never", it still happens on both.

I've a recording of bug occuring, would it help you?

The only reliable way to get rid of it, that I've found out is to scroll up till I'm pretty sure the page in question is unloaded, then scroll back down to it. OR I can close the reader and open it again, either method works.

**Comment by kolinrno** [comment]
@ColorCookie-dev 

yep, I still feel this bug in the latest version. disappointed :(

**Comment by VietAnh14** [comment]
I've found that this happen when 2 view holders are loading and the upper view holder done loading with bigger height than current height while the second one is still loading. The first will push the second out of the screen, making the `SubsamplingScaleImageView` being detached from window, then the `ViewScopeHolder` of the  `SubsamplingScaleImageView` will be cancelled, making it stuck at loading state.
I've removed `ViewScopeHolder` and this bug is gone.
